ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -29,22 +29,21 @@
         "id": "title"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    TRAIN_IMAGE = "us-docker.pkg.dev/vertex-ai/training/{}:latest".format(TRAIN_VERSION)
</pre><p>Why the change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>This is the new repository for pre-built container images based after the "Vertex AI" branding was introduced. See https://cloud.google.com/vertex-ai/docs/training/pre-built-containers</p><p> </p><p>I expect that the image you get from the old URI and the new one should be identical. (If not, that's a problem.)</p>